### ISSUE TITLE
Simulator not creating events and tags with correct parent

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#1311]
Small one. @Noojuno was trying to figure out why he couldn't find events or tags in the correct spot (i.e. `/events/` and `/tags/`. It sounds like they ones created by the simulator aren't being assigned the correct parent.

**Comment by sgstair** [comment]
Yeah they aren't being assigned a parent, will address this in the near future.
